modules/ui/edit_menu.js

```diff
@@ -226,7 +229,8 @@ export function uiEditMenu(context) {
         }
 
         var origin = geoVecAdd(anchorLoc, offset);
-
+        // adjusts for top menu height
+        origin[1] -= _verticalOffset;
```

Comment:
instead of the hardcoded offset, we could instead read the rendered height of the toolbar like this:

```suggestion
        // adjusts for top menu height
        var _verticalOffset = parseFloat(utilGetDimensions(d3_select('.top-toolbar-wrap'))[1]);
        origin[1] -= _verticalOffset;
```

This would then automatically continue to work if the toolbar size would change in the future.

Reply:
Great! that's changed!